lib/themes/base/tokens.ts

```diff
@@ -83,10 +83,14 @@ const makeColourGamut = () =>
 
 export const tokens: Tokens = {
 	breakpoints: {
-		mobile: 650,
-		tablet: 651,
-		desktop: 1001,
-		largeDesktop: 1201,
+		mobile: 0, // TODO: Check. Can this just be 0?
```

Comment:
mobile 0 works for me,  I think the 650 was there to be able to do mobile only max-width breakpoints in CSS land (calc support in media query is or at least was not global) but in the land of Treat we can easily get that from tablet